.github/workflows/deploy.yml

```diff
@@ -1,7 +1,7 @@
 name: Deploy
 on:
   push:
-    branches: [ main ]
+    branches: [ cd ]
```

Comment:
I'm sorry I'm not sure what `cd` stands for

Reply:
It's just the name of this branch, so that the deployment is triggered: otherwise, this branch is not deployed and we cannot test that it actually works 🙂 
“CD” stands for Continuous Deployment.